dev/start.sh

```diff
@@ -6,10 +6,10 @@ docker-compose build
 
 if [ ! -f data_key ]; then
 	echo "Generating data key"
-	docker-compose run --no-deps --rm --entrypoint possum possum data-key generate > data_key
+	docker-compose run --no-deps --rm --entrypoint conjur conjurctl data-key generate > data_key
```

Comment:
Tangentially, @kgilpin, do you think we could `openssl rand -base64 32` here instead? 
This line has always bothered me a bit.